layouts/index.html

```diff
@@ -1,14 +1,89 @@
+{{ $influxdbVersionV2 := .Site.Data.products.influxdb.latest }}
+{{ $influxdbVersionV1 := (index (last 1 (findRE `v1\.[0-9]{1,2}` (delimit .Site.Data.products.influxdb.versions " "))) 0) }}
+{{ $enterpriseVersion := .Site.Data.products.enterprise_influxdb.latest }}
+{{ $telegrafVersion := .Site.Data.products.telegraf.latest }}
+{{ $chronografVersion := .Site.Data.products.chronograf.latest }}
+{{ $kapacitorVersion := .Site.Data.products.kapacitor.latest }}
+
 {{ partial "header.html" . }}
+{{ partial "topnav.html" . }}
+
+<div class="home">
+
+  <div class="section hero">
+    {{ partial "svgs/homepage-hero.svg" }}
+    <div class="half">
+      <h2>InfluxDB Cloud & <br />InfluxDB {{ $influxdbVersionV2 }} OSS</h2>
+      <p>
+        The most powerful time series database designed to handle high write and query loads.
```

Comment:
We could split first sentence for emphasis:
The most powerful...Designed to handle